### PR TITLE
Medium support

### PR DESCRIPTION
**Pull request by danger9224**
Added Medium support to open instead of Facebook and Twitter.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-shepherd.artpi.net
```
Comment (artpi) [inline CNAME]: Please dont remove this domain, otherwise it will stop working, since its hosted by github pages :)

**Comment by artpi** [comment]
Thank you so much!
I am sorry that it took me so long :(

This is a perfect improvement , but please dont remove the CNAME domain. Please change that and it'd be good to go :)

File: CNAME (hunk 1 above)
Comment (danger9224) [reply CNAME]: Okay, I re-added it. :)

**Comment by danger9224** [comment]
Hey!
I don't know if you received a notification or something about the reply, but I re-added the CNAME. :)

**Pull request closed without merging**